roles/bootstrap/tasks/main.yml

```diff
@@ -104,11 +105,24 @@
     name: "{{ bootstrap_default_common_packages }}"
     state: present
 
+- name: Check if reboot required
+  ansible.builtin.stat:
+    path: /var/run/reboot-required
+  register: reboot_required_file
```

Comment:
Let's prefix the `reboot_required_file` variable to `bootstrap_reboot_required_file`. Ansibles variables scope is a bit too wide, so it's a good practice to prefix them with the roles name.